disnake/embeds.py

```diff
@@ -794,3 +800,64 @@ def _handle_resource(self, url: Optional[Any], file: File, *, key: _FileKey) ->
         else:
             self._files.pop(key, None)
             return str(url) if url is not None else None
+
+    def check_limits(self):
```

Comment:
Should have a mention somewhere in the docstring about a 6k overall limit. 

Additionally, the 6k limit is for all embeds in a message, so if a developer is sending multiple embeds they should check the sum of all embeds is less than 6k (if they're concerned)

For those interested, here's a short snippet to verify all embeds:
```py
# `embeds` is a list of embeds
total_len = sum(map(len, embeds))
```